crates/auxmos/src/gas.rs

```diff
@@ -30,30 +33,27 @@ static GAS_MIXTURES: RwLock<Option<Vec<RwLock<Mixture>>>> = const_rwlock(None);
 
 static NEXT_GAS_IDS: RwLock<Option<Vec<usize>>> = const_rwlock(None);
 
-static mut REGISTERED_GAS_MIXES: Option<HashSet<u32, FxBuildHasher>> = None;
+static REGISTERED_GAS_MIXES: Mutex<Option<HashSet<u32, FxBuildHasher>>> = const_mutex(None);
```

Comment:
this can actually be made a thread_local, since it's only actually used in one thread